### PR TITLE
update container-runtime.md

### PR DESCRIPTION
**Pull request by dejanu**
Update container runtime description

### CODE REVIEW

File: content/en/docs/reference/glossary/container-runtime.md [hunk 1]
```diff
@@ -11,7 +11,8 @@ tags:
 - fundamental
 - workload
 ---
- The container runtime is the software that is responsible for running containers.
+ Kubernetes doesn’t know how to run containers it relies on the container runtime component. 
+ The container runtime is a fundamental component that is responsible for managing the execution and lifecycle of containers.
```
Comment (windsonsea) [inline content/en/docs/reference/glossary/container-runtime.md:15]:
```suggestion
 A fundamental component that empowers Kubernetes to run containers effectively.
 Its responsibility lies in managing the execution and lifecycle of containers within the Kubernetes environment.
```
Only for your reference.

File: content/en/docs/reference/glossary/container-runtime.md [hunk 2]
```diff
@@ -11,7 +11,8 @@ tags:
 - fundamental
 - workload
 ---
- The container runtime is the software that is responsible for running containers.
+ A fundamental component that empowers Kubernetes to run containers effectively.
+ Its responsibility lies in managing the execution and lifecycle of containers within the Kubernetes environment.
```
Comment (divya-mohan0209) [inline content/en/docs/reference/glossary/container-runtime.md:15]: Making this a little more idiomatic,

```suggestion
 It is responsible for managing the execution and lifecycle of containers within the Kubernetes environment.
```

**Review by divya-mohan0209 — COMMENTED** [review]
Suggested a more idiomatic phrasing of the content.

**Pull request merged**